mypy/errors.py

```diff
@@ -235,7 +240,7 @@ def copy(self) -> 'Errors':
         return new
 
     def total_errors(self) -> int:
-        return sum(len(errs) for errs in self.error_info_map.values())
+        return self.total_error_count
```

Comment:
This is only used in one place, where we check that the error count before and after some operation is the same. I wonder if we can do something simpler there.

Reply:
Tried something a little more complex but which doesn't require maintaining a state that might diverge